packages/stardog-graphql-language-server/src/GraphQlLanguageServer.ts

```diff
@@ -188,21 +188,39 @@ export class GraphQlLanguageServer extends AbstractLanguageServer<
         update.relationshipBindings || this.relationshipBindings;
       this.relationshipCompletionItems = this.buildCompletionItemsFromData(
         this.namespaceMap,
-        this.relationshipBindings.map((binding) => ({
-          iri: binding.relationship.value,
-          count: binding.count.value,
-        })),
+        this.relationshipBindings
+          .map((binding) => ({
+            iri:
+              binding &&
+              binding.relationship &&
+              binding.relationship.value !== undefined
+                ? binding.relationship.value
+                : undefined,
+            count:
+              binding && binding.count && binding.count.value !== undefined
+                ? binding.count.value
+                : undefined,
```

Comment:
these can just be `binding && binding.relationship && binding.relationship.value`

Reply:
37f579739d3fbcbe8efb252c2cc95b51fb8da1bd

Reply:
same with the counts!